packages/drivers/routerlicious-driver/src/urlUtils.ts

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+import URLParse = require("url-parse");
```

Comment:
Do we really need to import using require here?

Reply:
Ah, looks like you're right, we don't. VSCode was telling me that it had to be imported via require because of the esModuleInterop flag being false, but it is in fact true. Restarting VSCode resolved the issue. Thanks for the callout!